xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml

```diff
@@ -862,6 +869,9 @@ define('editInPlace', [
     insert: function(element) {
       this._getActionButtons().find('.hidden.extra').append(element);
     },
+    append: function(element) {
```

Comment:
To be honest I added that one because I'm using `$('#form').append(node)` and when transforming the code to use either `$('#form')` or `fakeform` I noticed that `insert` does not exist in jquery so felt easier to just create this append method there. So if that's ok to you I might just add a comment to specify that only the first element appending is supported there.